### DISCUSSION TITLE
qiskit not supported in colab

### DISCUSSION THREAD

**Discussion by Snuag** [NVIDIA/cuQuantum#57] [Q&A] [answered]
HELLO! 
I am trying to run the QAOA file  from https://github.com/NVIDIA/cuQuantum/blob/main/python/samples/cutensornet/circuit_converter/qiskit_advanced.ipynb
but i get the following error saying 'qiskit not supported'. I have installed the dependencies mentioned by @leofang 
![image](https://github.com/NVIDIA/cuQuantum/assets/100287263/495d30b2-2b4d-4bb4-afee-c6fd1a876ad4)

Can you please tell me where I am going wrong!?

Here are the initial installations made:
! nvcc --version    - 11.8
!pip install cuquantum
!pip install --upgrade cuquantum
!pip install cuquantum-python
!pip install cuquantum-python-cu11
!pip install cuquantum-python-cu11 cupy-cuda11x
!pip install qiskit
import cuquantum

**Comment by leofang** [comment]
Interesting, could you share the output of `qiskit.__qiskit_version__`?

**Reply by Snuag** [reply]
Hi @leofang . Here is the snippet of output 
![image](https://github.com/NVIDIA/cuQuantum/assets/100287263/5a0cbd18-2fc6-4758-b235-d42874bf26ac)

**Reply by leofang** [reply]
Could you print `qiskit.__qiskit_version__` not `qiskit.__version__` so that we can see all Qiskit components' versions?

**Comment by Snuag** [comment, marked as answer]
This was the output :
'qiskit-ibmq-provider': '0.20.2', 'qiskit': '0.43.0', 'qiskit-nature': None, 'qiskit-finance': None, 'qiskit-optimization': None, 'qiskit-machine-learning': None}
And I have installed all of the above dependencies,,, and now its working! Thank you @leofang

**Reply by leofang** [reply]
Thanks, @Snuag. TBH this is surprising because we shouldn't need anything other than Qiskit-Terra that offers the `QuantumCircuit` class. But glad to know it's working for you.
